strikecircle/tasks.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from .env import env
+from logging import getLogger
+from django.core.serializers import serialize
+from strikecircle.models import Pledge, StrikeCircle
+
+from airtable import Airtable
+from background_task import background
+
+
+logger = getLogger(__name__)
+
+
+API_KEY = env("AIRTABLE_API_KEY")
+BASE_KEY = env("AIRTABLE_BASE_KEY")
+
+
+# TODO: Deploy supervisord in prod to run tasks https://django-background-tasks.readthedocs.io/en/latest/#running-tasks
+
+
+@background(schedule=10, queue="airtable-export")
+def export_strike_circle_to_airtable(strike_circle_id):
+	"""
+	type: (int) -> Dict[str, Any]
```

Comment:
Thanks for including the types here. I'm not putting the responsibility for this on you, but we should probably start adding [type hints](https://docs.python.org/3/library/typing.html) to the project at some point.